src/histolab/data/_registry.py

```diff
@@ -0,0 +1,27 @@
+# flake8: noqa
+
+# in legacy datasets we need to put our sample data within the data dir
+legacy_datasets = ["cmu_small_region.svs"]
+
+# Registry of datafiles that can be downloaded along with their SHA256 hashes
+# To generate the SHA256 hash, use the command
+# openssl sha256 filename
+registry = {
+    "data/cmu_small_region.svs": "ed92d5a9f2e86df67640d6f92ce3e231419ce127131697fbbce42ad5e002c8a7",
+    "aperio/JP2K-33003-1.svs": "6205ccf75a8fa6c32df7c5c04b7377398971a490fb6b320d50d91f7ba6a0e6fd",
+    "aperio/JP2K-33003-2.svs": "1a13cef86b55b51127cebd94a1f6069f7de494c98e3e708640d1ce7181d9e3fd",
+    "tcga/breast/9c960533-2e58-4e54-97b2-8454dfb4b8c8": "03f542afa2d70224d594b2cca33b99977a5c0e41b1a8d03471ab3cf62ea3c4b3",
```

Comment:
maybe we could use the TCGA "project name" such as TCGA-BRCA instead of the general "breast"... being more specific could help the scientific community

Reply:
this is one of the question i'm gonna leave you both @alessiamarcolini and @nicolebussola. This file is just a registry the users will get the svs using the function name in the init of this module. So the important thing is that the meaningful name should be in there, isn't it?